### ISSUE TITLE
Rectangle stopped working after updating app yesterday

### ISSUE THREAD

**Issue by earthling97** [rxhanson/Rectangle#1337]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: OS 14.3.1
Rectangle version: .74, .75, .76
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

I have had Rectangle for a few weeks and really like it. Been using it every day. I updated to .76 when prompted that a new version was available, and then the menu bar icon disappeared. Tried everything I could think of (restart Mac, shut down Mac,  uninstall and reinstall Rect., downloaded previous versions as above) with no luck. I tried disabling and re-enabling in Privacy/Security/Accessibility system settings. No luck.  Strangely, when I do Option-Command-Escape to see open apps, Rectangle doesn't show up there.  Yet when I try to uninstall by moving the application icon to the trash it says cannot do because the app is open. So I use CleanMyMac to uninstall.  It seems the application isn't opening properly. Any ideas??? Would love to get back to using it.  Thanks

**Comment by rxhanson** [comment]
Perhaps something happened mid-update that broke the app? (I've never encountered that before)

Cmd+option+escape does not show background/agent apps like Rectangle - you'd have to go into Activity Monitor and kill the app from there (or from Terminal) prior to deleting the app. 

Try deleting the defaults with this terminal command:
defaults delete com.knollsoft.Rectangle

Then reinstall a fresh download of v0.76.

**Comment by earthling97** [comment]
I found out the problem! I was trying to think of any software I had installed prior to updating Rectangle. The only thing that I had installed was CleanMyMac. So I uninstalled that (after deleting Rectangle from the Activity Monitor suggested), I then restarted my computer--just because--and then downloaded a fresh copy of Rectangle. It works now! So there's apparently some kind of conflict with CleanMyMac. 

I'm going to reinstall that and see what happens. But I thought you would like to know about this conflict I encountered.

Thank you for being accessible and producing a great product. 

Chris
